src/index.ts

```diff
@@ -56,15 +56,15 @@ export const moxci = async (targetPath: string, options: Options) => {
 
   // Github
   if (GITHUB_TOKEN) {
-    const pullRequestId = CIRCLE_PULL_REQUEST.split("/").pop();
+    const pullRequestId = Number(CIRCLE_PULL_REQUEST.split("/").pop());
```

Comment:
`issue_number` argument of `octokit.issues.createComment` expects a number type, so converting to a number type here.